content/news/XXXX-XX-XX-mixxx-2.4.2-released.md

```diff
@@ -0,0 +1,167 @@
+title: Mixxx 2.4.2 Release Announcement
+authors: Evelynne Veys
+tags: update, 2.4.2
+comments: yes
+status: draft
+
+#### Winter and Summer are coming
+
+Contrasts.
+As the leaves fall and the first snowflakes start drifting down in the North, while the southern skies start warming up
+with summer sun, we are thrilled to finally release Mixxx 2.4.2! Finally because a bit later than planned, perfection
+can take some time. So we are proud to present you this release, a major bugfix-release for the 2.4 series before
+stepping over to a new number.
+Don't be scared, development won't hibernate.
+
+#### Testing and translating
+
+A special thank you to everyone who takes the time to test alfa and beta versions, to who report issues and proposes and to those
+who propose features and improvements. Your feedback is invaluable.
+A lot of new users and contributors have joined our community the past months, we'd like to welcome them all, spin it up,
+your energy and fresh ideas have already left a mark.
+Everybody can contribute to Mixxx. Did you remark an error on the website, an incomplete subject in the manual, an error in translation?
+[Contribute](https://mixxx.org/get-involved/).
+
+#### Treasure hunt
+
+When you dive in the code to add a feature or when you are looking for a function when want to customize a mapping,
+you can discover small little snippets, real pearls, the intensive work of someone to make a function smooth, to let a conversio
+work some milliseconds faster, to offer a conversion of a value to another without having to write it all over again.
+For all these little snippets, for all these forgotton pearls: thank you to all who ever contributed.
+
+#### What's in the 2.4.2-bag?
+
+All new and shiny: [the complete changelog 2.4.2](https://github.com/mixxxdj/mixxx/blob/2.4/CHANGELOG.md)
+This new stable release can be downloaded from the [Download page](https://mixxx.org/download/).
+
+Problems that were solved:
+* Handle not supported files when dragging to waveforms and spinnies

```

Comment:
lists are usually surrounded by blank lines. Does that fix the issue?
```suggestion
Problems that were solved:

* Handle not supported files when dragging to waveforms and spinnies
```

Reply:
Thank you very much, I feel so stupid... but I 'll never forget

Reply:
No worries ;)